### ISSUE TITLE
Practice 6 - simple_local_planner - wrong closest object distance and term consistency

### ISSUE THREAD

**Issue by geopimik** [Karljohan99/autoware_mini_practice#9]
Currently, your simple local planner would stop before the obstacle like this:

![image](https://github.com/Karljohan99/autoware_mini_practice/assets/33843059/dd081dc7-4cc6-4dd4-b1d3-41c68262d4bb)

It is roughly 4m before it as needed, but the `Closest obj dist` is showing 7.69m. So it is wrong here:
https://github.com/Karljohan99/autoware_mini_practice/blob/a09abb4acfa7dc931d7fc454a4906f6bcbae5884/practice_6/nodes/planning/local/simple_local_planner.py#L183

It is related how you interpret the following terms. I'll bring them here with our definitions:
   - `object_distances` - from base_link to closest point of object
   - `object_braking_distances` - additional object specific braking distance (how close to an object we want to stop)

It is up to you how you define them, but they need consistency. For example:
   - Currently for **goal point** you have a line in code, but additional parameter: `self.braking_safety_distance_goal` is not used. Since it is a parameter and someday we might change it to 10m, it would not change the behaviour. Need to fix it.
https://github.com/Karljohan99/autoware_mini_practice/blob/a09abb4acfa7dc931d7fc454a4906f6bcbae5884/practice_6/nodes/planning/local/simple_local_planner.py#L168
   - For **obstacle** you do it like this:
https://github.com/Karljohan99/autoware_mini_practice/blob/a09abb4acfa7dc931d7fc454a4906f6bcbae5884/practice_6/nodes/planning/local/simple_local_planner.py#L160-L161

#### Summary:
* Make the terms clear and use them consistently
* Calculate the `closest_object_distance` correctly

**Comment by Karljohan99** [comment]
Made the changes in 00e500c. I hope that the `closest_object_distance` is now calculated correctly. It seems to me that the value for `self.current_pose_to_car_front` is too large and therefore I couldn't get the `Closest obj dist` exactly right. The measuring tool also behaved weirdly. If I measured the distance from the `base_link` exactly to the front of the car, it said around 4 meters, but if I moved a little further with my mouse, it said less than 3.5 meters.
